content/200-concepts/100-components/02-prisma-client/053-middleware/index.mdx

```diff
@@ -43,28 +45,71 @@ Use cases for middleware include:
 
 <Subsections />
 
-## Running order
+## Where to add middleware
+
+Add Prisma middleware **outside the context of the request handler**. The following example demonstrates where to add Prisma middleware in the context of an Express app:
```

Comment:
Maybe add an explanation why? "Otherwise each request would add another instance of the middleware to the stack - which is probably not what you want."